src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/model/in/ExceptionRecord.java

```diff
@@ -18,6 +19,7 @@
     public final List<OcrDataField> ocrDataFields;
 
     public ExceptionRecord(
+        @JsonProperty("id") String id,
```

Comment:
Let's discuss it with the team. It might be better if the orchestrator did all the referencing from documents, instead of services.

Reply:
Discussed 👍